src/shared/components/palette/case-flag/write-case-flag-field.component.ts

```diff
@@ -5,6 +5,7 @@ import { CaseField, ErrorMessage } from '../../../domain';
 import { FieldsUtils } from '../../../services/fields';
 import { CaseEditPageComponent } from '../../case-editor/case-edit-page/case-edit-page.component';
 import { AbstractFieldWriteComponent } from '../base-field/abstract-field-write.component';
+import { UpdateFlagComponent } from './components';
```

Comment:
@johnbenjamin-hmcts - just a minor one, please change import to fully qualified path

Reply:
@Daniel-Lam - I have updated the code.